packages/app/src/cli/models/app/app.test.ts

```diff
@@ -48,12 +47,6 @@ const CORRECT_CURRENT_APP_SCHEMA: CurrentAppConfiguration = {
     subpath: 'https://google.com',
     prefix: 'https://google.com',
   },
-  pos: {
-    embedded: false,
-  },
-  app_preferences: {
-    url: 'https://google.com',
-  },
```

Comment:
Why removed these?

Reply:
They're not part of the config required by CLI -- the values were only set here -- and I figured the right shape for this object is to be minimal